src/main.ts

```diff
@@ -15,6 +16,18 @@ export async function run() {
     let version = core.getInput('node-version');
     if (!version) {
       version = core.getInput('version');
+
+      if (!version) {
+        const versionFile = core.getInput('node-version-file');
+
+        if (!!versionFile) {
+          const versionFilePath = path.join(__dirname, '..', versionFile);
```

Comment:
Could you please change this line from `const versionFilePath = path.join(__dirname, '..', versionFile);` to `const versionFilePath = path.join(process.env.GITHUB_WORKSPACE!, versionFile);`